CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Not released yet
 
+* Allows repack without Castor logo e.g. `castor repack --hide-logo`
```

Comment:
I think we could do better. `--no-logo` or `--logo=path/to/file.txt` (with negatable option)